### PR TITLE
Feb 2 2018 kenman345

### PR DESCRIPTION
**Pull request by kenman345**
The rest of the requests are a bit of questions that need reviewing but this should clear some things up

### CODE REVIEW

**Comment by kenman345** [comment]
fixes #517

File: _data/mining.yml [hunk 1]
```diff
@@ -35,13 +35,35 @@ websites:
...
       othercrypto: Yes
+      
+    - name: Taimine
+      url: https://taimine.com
+      img: taimine.png
+      twitter: taimineminer
+      facebook: taimineindustries
+      email_address: service@taimine.com
+      region: as
+      city: all of asia
```
Comment (joshmcintyre) [inline _data/mining.yml]: Can probably just omit the city tag here, but I'm not that picky

Comment (kenman345) [reply _data/mining.yml]: done

Comment (kenman345) [reply _data/mining.yml]: @joshmcintyre good catch, Fixed and ready to go!

After we merge please take a look over the rest of the issues. I would've done them but I wasnt sure the categories they should fall into.

**Pull request merged**